### PR TITLE
BZ1991850: Adding ptp4lConf parameter to PtpConfig.yaml

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.8

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=1991850

Link to docs preview: http://file.emea.redhat.com/amolnar/BZ1991850/networking/configuring-ptp.html#configuring-linuxptp_configuring-ptp

Additional information: This parameter is present in main and 4.9+.

### CODE REVIEW

**Comment by djuran** [comment]
LGTM

**Comment by obochan-rh** [comment]
LGTM

File: modules/nw-ptp-configuring-linuxptp.adoc [hunk 1]
```diff
@@ -46,12 +47,13 @@ spec:
...
 <7> Specify system config options for the `phc2sys` service, for example `-a -r`.
-<8> Specify an array of one or more `recommend` objects which define rules on how the `profile` should be applied to nodes.
-<9> Specify the `profile` object name defined in the `profile` section.
-<10> Specify the `priority` with an integer value between `0` and `99`. A larger number gets lower priority, so a priority of `99` is lower than a priority of `10`. If a node can be matched with multiple profiles according to rules defined in the `match` field, the profile with the higher priority will be applied to that node.
-<11> Specify `match` rules with `nodeLabel` or `nodeName`.
-<12> Specify `nodeLabel` with the `key` of `node.Labels` from the node object by using the `oc get nodes --show-labels` command.
-<13> Specify `nodeName` with `node.Name` from the node object by using the `oc get nodes` command.
+<8> Specify a string that contains the configuration to replace the default /etc/ptp4l.conf file. To use the default configuration, leave the field empty.
+<9> Specify an array of one or more `recommend` objects which define rules on how the `profile` should be applied to nodes.
```
Comment (tmalove) [inline modules/nw-ptp-configuring-linuxptp.adoc:51]:
```suggestion
<9> Specify an array of one or more `recommend` objects, which define rules on how the `profile` should be applied to nodes.
```

Comment (tmalove) [reply modules/nw-ptp-configuring-linuxptp.adoc:51]: See ISG for reference: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-with-nonrestrictive-clauses-and-phrases

**Pull request merged**